src/client/dfs/dfs.c

```diff
@@ -2147,6 +2147,54 @@ dfs_disconnect(dfs_t *dfs)
 	return rc;
 }
 
+static int
+get_uid_gid_from_ownership(daos_handle_t coh, dfs_t *dfs)
+{
+	daos_prop_t            *owner_props;
+	struct daos_prop_entry *entry;
+	int                     rc = 0;
+
+	owner_props = daos_prop_alloc(2);
+	if (owner_props == NULL)
+		return ENOMEM;
+
+	owner_props->dpp_entries[0].dpe_type = DAOS_PROP_CO_OWNER;
+	owner_props->dpp_entries[1].dpe_type = DAOS_PROP_CO_OWNER_GROUP;
+
+	rc = daos_cont_query(coh, NULL, owner_props, NULL);
+	if (rc) {
+		D_ERROR("daos_cont_query() failed, " DF_RC "\n", DP_RC(rc));
```

Comment:
Side note - `" DF_RC "` or `"DF_RC"`? It seems we use both in the code base, but `"DF_RC"` is more common

Reply:
clang-format only supports the former which is a shame as the majority of our code uses the latter.